changedetectionio/__init__.py

```diff
@@ -628,9 +624,10 @@ def import_page():
             urls = request.values.get('urls').split("\n")
             for url in urls:
                 url = url.strip()
+                url, *tags = url.split(" ")
```

Comment:
whats the use of the `*` here?

Reply:
`*` packs all remaining outputs into a list, ie. if you have `https://google.com search other`, it will put search and other into the `tags` list (likewise an empty list for no tags). If you don't do this, it will complain that there may be too many/too few items for all cases other than there being one single tag.